lib/ansible/modules/cloud/digital_ocean/_digital_ocean_sshkey.py

```diff
@@ -19,7 +19,6 @@
                     'status': ['preview'],
```

Comment:
You need to change preview to "deprecated".

[Ansible metadata-block](https://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#ansible-metadata-block)

needs_revision